xml/favicons.xml

```diff
@@ -168,8 +173,10 @@
     <description>Metasploit Pro</description>
     <example>08ff173efec0750dd29ac7f44d972427</example>
     <param pos="0" name="service.vendor" value="Rapid7"/>
-    <param pos="0" name="service.product" value="Metasploit Pro"/>
+    <param pos="0" name="service.product" value="Metasploit"/>
+    <param pos="0" name="service.edition" value="Pro"/>
```

Comment:
I made this change because it's in-line with how we handle editions elsewhere and it allows autogeneration of the CPE.  I can change this back and add a mapping if folks disagree.

As a note, here is how NIST generates the CPE for Pro
```
cpe:2.3:a:rapid7:metasploit:4.16.0:20190722:*:*:pro:*:*:*
```